src/plugins/intel_gna/src/transformations/pwl/configuration_exp.cpp

```diff
@@ -0,0 +1,73 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "configuration_exp.hpp"
+
+#include "common.hpp"
+#include "ie_common.h"
+#include "openvino/opsets/opset11.hpp"
+#include "openvino/pass/pattern/op/wrap_type.hpp"
+
+namespace ov {
+namespace intel_gna {
+namespace pass {
+namespace pwl {
+
+const std::string ConfigurationExp::kActivationName = "Exp";
+const double ConfigurationExp::kLowerBound = -std::log2(std::numeric_limits<int16_t>::max());
```

Comment:
Could you clarify why such values for kLowerBound and kUpperBound were chosen? Using log2 and log10 looks like an accident to me, please add a comment if it's intentional

Reply:
I am not sure.
this parameters where used in previous version of algorithm.
I suspect that this is related to limitations when conversion from flaots to integers happening further in backend.